addons/ai/XEH_preInit.sqf

```diff
@@ -6,16 +6,12 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
-// Fix rotating garrisoned units
+GVAR(skills) = [false, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, true, true, true];
```

Comment:
Technically, magic numbers (I typically define them not being 0 or 1).

Reply:
I don't see how this could be improved.